src/app/service/poap/POAPService.ts

```diff
@@ -49,7 +49,8 @@ const POAPService = {
 			const poapOrganizer: GuildMember = await poapGuild.members.fetch(expiredEvent.discordUserId);
 			EndPOAP(poapOrganizer, platform).catch((e) => {
 				if (e instanceof ValidationError) {
-					poapOrganizer.send({ content: `${e?.message}` }).catch(Log.error);
+					// poapOrganizer.send({ content: `${e?.message}` }).catch(Log.error);
```

Comment:
Is this on purpose? no error message back to the user

Reply:
yes, because discord might see this as a random DM (whenever the poap event ends) and flag it unfortunately...